doc/community/developer-guide.md

```diff
@@ -19,10 +19,28 @@ myst:
 
 # Developer guide
 
+Before continuing, ensure you have a working [development environment.](https://ploomber-contributing.readthedocs.io/en/latest/contributing/setup.html)
```

Comment:
I think that this entire developer-guide.md file should be renamed and moved into a "Testing" -> "JupySQL" section within the [contributing portal](https://ploomber-contributing.readthedocs.io/en/latest/intro.html)

I also found it a bit difficult to navigate to the contribution docs from the main website, and would suggest just making the "Developer guide" link pictured below directly go to [Contributing](https://ploomber-contributing.readthedocs.io/en/latest/intro.html) since that covers all development instructions for all of your projects

Reply:
![Screen Shot 2023-04-14 at 1 17 08 PM](https://user-images.githubusercontent.com/21978330/232113079-3ed582a1-0d2f-4612-8288-57eabb11b0b8.png)